ugvc/vcfbed/variant_annotation.py

```diff
@@ -156,18 +156,36 @@ def process_vcf(
             tmp_output_paths = []
             sp = SimplePipeline(0, 100)
             commands = []
+            variant_counts = {}  # Dictionary to store variant counts per contig
+            skipped_contigs = []
             ugvc_path = os.path.dirname(os.path.dirname(__file__))
             for contig in contigs:
                 try:
                     next(input_variant_file.fetch(contig))  # to raise StopIteration if contig is empty
-                    out_per_contig = os.path.join(out_dir, contig + ".vcf.gz")
-                    commands.append(
-                        f"python {ugvc_path} annotate_contig --vcf_in {input_path} --vcf_out {out_per_contig} "
-                        f"--annotators_pickle {annotators_pickle} --contig {contig} --chunk_size {chunk_size}"
-                    )
-                    tmp_output_paths.append(out_per_contig)
-                except StopIteration:
-                    pass
+                    variant_count = 1
+                    # Iterate through variants in the contig and count them
+                    # for _ in input_variant_file.fetch(contig):
+                    #     variant_count += 1
+
+                    if variant_count > 0:
```

Comment:
I don't understand these two lines, seems to me like they do nothing:
```
variant_count = 1
if variant_count > 0:
```

What was wrong with the previous version?